numpy/lib/nanfunctions.py

```diff
@@ -1190,8 +1229,8 @@ def _nanquantile_1d(arr1d, q, overwrite_input=False, interpolation='linear'):
     Private function for rank 1 arrays. Compute quantile ignoring NaNs.
     See nanpercentile for parameter usage
     """
-    arr1d, overwrite_input = _remove_nan_1d(arr1d,
-        overwrite_input=overwrite_input)
+    arr1d, overwrite_input = _remove_nan_1d(
+        arr1d, overwrite_input=overwrite_input)
```

Comment:
I agree with this wrapping, but would prefer to see it left to another PR

Reply:
Done.

Reply:
It's showing in the diff just fine - you just did it in two places, and only reverted it in one

Reply:
Ahh, my bad. Fixed.